### DISCUSSION TITLE
What user does SerpBear run under in Docker?

### DISCUSSION THREAD

**Discussion by Stitch10925** [towfiqi/serpbear#265] [Q&A]
I am running SerpBear in Docker Swarm using NFS as volume store. When starting SerpBear I get an access denied on the "/app/data" folder. What are the permissions I need to set? What user is SerpBear running as?

**Comment by abdessalaam** [comment]
I've added to docker-compose.yml:  

```yml
user: 1000:1000
```  

and them problem seems solved!

**Comment by Stitch10925** [comment]
Yes, that seems to do the trick. Thanks!
